include/extensions.h

```diff
@@ -11,6 +11,8 @@ struct background_config {
         wlc_resource surface;
         // we need the wl_resource of the surface in the destructor
         struct wl_resource *wl_surface_res;
+        // used to determine if client is a background
+	struct wl_client *client;
```

Comment:
If you find a file that doesn't conform to the rules, please reformat it as necessary.